oms-kemp-applicationdelivery-solution/azuredeploy.json

```diff
@@ -0,0 +1,906 @@
+                "alertTresholdValue": 0,
+                "operator": "gt",
+                "alertThrottleInMinutes": 0,
+                "scheduleIntervalInMinutes": 60,
+                "scheduleQueryTimeSpan": 60,
+                "scheduleType":"Normal",
+                "scheduleTypeSpecified": true,
+                "nearRealTime": false,
+                "version":1
+            }
+        ],
+        "alertMetricArray": [
+            {
+                "alertName": "KEMP - Virtual Server (VS) not operational",
+                "description": "The VS status is not Up this means the service provided by this VS is not available",
+                "severity": "Critical",
+                "enabled": true,                
+                "searchName": "Alert - Virtual Server (VS) not operational",
+                "searchCategory": "[variables('savedSearchesCategory')]",
+                "query": "Type:KempStatus_CL (servertype_s=vs OR servertype_s=subvs) enabled_s=Y AND (status_s != Up AND status_s != Redirect) | measure count() by Computer, name_s interval 10minutes",
+                "alertTresholdValue": 0,
+                "operator": "gt",
+                "alertThrottleInMinutes": 0,
+                "scheduleIntervalInMinutes": 5,
+                "scheduleQueryTimeSpan": 15,
+                "triggerCondition":"Total",
+                "triggerOperator": "gt",
+                "triggerValue": 0,
+                "scheduleType":"Normal",
+                "scheduleTypeSpecified": true,
+                "nearRealTime": false,
+                "version": 1
+            },
+            {
+                "alertName": "KEMP - TPS approaching the device capacity",
+                "description": "The current load in TPS is approaching the device limit. Take action to avoid service disruption",
+                "severity": "Critical",
+                "enabled": false,
+                "searchName": "Alert - TPS approaching device limit",
+                "searchCategory": "[variables('savedSearchesCategory')]",
+                "query": "Type:Perf ObjectName=KempLM-TPS CounterName=\"Total TPS\" | measure max(CounterValue) by Computer interval 15MINUTE",
+                "alertTresholdValue": 0,
+                "operator": "gt",
+                "alertThrottleInMinutes": 0,
+                "scheduleIntervalInMinutes": 15,
+                "scheduleQueryTimeSpan": 60,
+                "triggerCondition":"Total",
+                "triggerOperator": "gt",
+                "triggerValue": 190,
+                "scheduleType":"Normal",
+                "scheduleTypeSpecified": true,
+                "nearRealTime": false,
+                "version": 1
+            }
+        ]        
+    },
+    "resources": [
+        {
+            "name": "[parameters('omslogAnalyticsWorkspaceName')]",
+            "type": "Microsoft.OperationalInsights/workspaces",
+            "apiVersion": "[variables('omsLogAnalyticsAPIVersion')]",
```

Comment:
please set the apiVersion directly, for betting tooling references and best practices, instead of referencing a variable